ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/DdlDmlResponseHandlers.java

```diff
@@ -68,6 +72,14 @@ private static boolean isCommandStatusEntity(final JsonObject ksqlEntity) {
         && ksqlEntity.getJsonObject("commandStatus") != null;
   }
 
+  private static boolean isIfNotExistsWarning(final JsonObject ksqlEntity) {
+    return ksqlEntity.getString("statementText") != null
+        && ksqlEntity.getString("statementText").contains("IF NOT EXISTS")
+        && ksqlEntity.getString("message") != null
+        && ksqlEntity.getString("message").contains("with the same name already exists.")
+        && ksqlEntity.getString("warnings") != null;
```

Comment:
```
{"@type":"warning_entity","statementText":"CREATE STREAM IF NOT EXISTS high_value_stock_trades\n    WITH (KAFKA_TOPIC='high_value_stock_trades') AS\nSELECT U.ID USERID,\n    U.REGIONID REGION,\n    T.SIDE SIDE,\n    T.QUANTITY QUANTITY,\n    T.SYMBOL SYMBOL,\n    T.PRICE PRICE,\n    T.ACCOUNT ACCOUNT\nFROM STOCKTRADES T\nINNER JOIN USERS U ON (T.USERID = U.ID)\nWHERE ((T.PRICE > 500) AND (T.QUANTITY > 2500))\nEMIT CHANGES;","message":"Cannot add stream `HIGH_VALUE_STOCK_TRADES`: A stream with the same name already exists.","warnings":[]}
```

Checking for a warnings field does not actually check if the entity is a `WarningEntity`

Reply:
```{"statementText":"CREATE STREAM IF NOT EXISTS ","message":"Cannot add stream `HIGH_VALUE_STOCK_TRADES`: A stream with the same name already exists.","warnings":[]}